### PR TITLE
Added subcommand help to poetry run happy command

### PR DESCRIPTION
**Pull request by Kd-Here**
Added subcommand help to poetry run happy command #44

### CODE REVIEW

File: happy/main.py [hunk 1]
```diff
@@ -66,6 +66,8 @@ def cli() -> None:
     )
 
     args = parser.parse_args(argv[1:])
+    parser.print_help()
```
Comment (Kd-Here) [inline happy/main.py:69]: Is this wrong place for printhelp 
Should i cretae if else statement?

**Comment by TechWiz-3** [comment]
Perfect thanks for your work

**Comment by Kd-Here** [comment]
Thanks TechWiz for your guidance and help throughout the task.
Any feedback for begineer how to contribute more.

**Comment by TechWiz-3** [comment]
Browse GitHub a lot. I personally like to use my CLI tool [scout](https://github.com/TechWiz-3/scout) to find new projects. But even without scout, simply explore :)

Just a few little tips:
1. make sure to test your changes before submitting a PR
2. look into what `poetry` is, as it was clear you were a little confused about what it does. Poetry simply manages dependencies and the developer environment for a project so that developers can test their changes in an isolated environment
3. use projects to explore libraries, for example, `happy` uses argparse and [rich](rich.readthedocs.io/) a lot, so you can use this opportunity to further research these libraries and understand how they work

feel free to ask me any questions :)

**Pull request merged**